adabot/lib/common_funcs.py

```diff
@@ -91,22 +91,31 @@ def parse_gitmodules(input_text):
         results.append((submodule_name, submodule_variables))
     return results
 
-def get_bundle_submodules():
-    """Query Adafruit_CircuitPython_Bundle repository for all the submodules
-    (i.e. modules included inside) and return a list of the found submodules.
+def get_bundle_submodules(bundle):
+    """Query Adafruit_CircuitPython_Bundle or CircuitPython_Community_Bundle
+    repository for all the submodules (i.e. modules included inside) and
+    return a list of the found submodules.
     Each list item is a 2-tuple of submodule name and a dict of submodule
     variables including 'path' (location of submodule in bundle) and
     'url' (URL to git repository with submodule contents).
     """
     # Assume the bundle repository is public and get the .gitmodules file
     # without any authentication or Github API usage.  Also assumes the
     # master branch of the bundle is the canonical source of the bundle release.
-    result = requests.get('https://raw.githubusercontent.com/adafruit/Adafruit_CircuitPython_Bundle/master/.gitmodules',
-                          timeout=15)
-    if result.status_code != 200:
-        #output_handler("Failed to access bundle .gitmodules file from GitHub!", quiet=True)
-        raise RuntimeError('Failed to access bundle .gitmodules file from GitHub!')
-    return parse_gitmodules(result.text)
+    if bundle == "Adafruit_CircuitPython_Bundle":
```

Comment:
How about making bundle `adafruit/Adafruit_CircuitPython_Bundle` and `adafruit/CircuitPython_Community_Bundle`? That way you can put it into the url and don't need an if statement. (It'll also work for other bundles then.)

Reply:
Changes were made according to suggestions, see https://github.com/adafruit/adabot/pull/218/commits/57d59aebac6c3f1abc46b3bc5f9f8ce5da7832fc. Thanks for reviewing